src/js/media-time-range.js

```diff
@@ -79,11 +75,10 @@ template.innerHTML = `
 
     media-preview-time-display,
     ::slotted(media-preview-time-display) {
-      color: unset;
```

Comment:
- Removed for consistency

Reply:
we have to be careful here, should be tested with Mux player's changeable colors